ffcx/codegeneration/ufcx.h

```diff
@@ -162,7 +162,7 @@ extern "C"
     bool needs_facet_permutations;
 
     /// Get the coordinate element associated with the geometry of the mesh.
-    long int coordinate_element;
+    unsigned long int coordinate_element;
```

Comment:
I'm getting a lot of `warning: integer constant is so large that it is unsigned` when compiling the generated code for this, but adjusting the type doesn't seem to make any difference

Reply:
Adding `u` to the end of each literal int removed this warning